service/src/main/resources/db/changelog.xml

```diff
@@ -1,5 +1,21 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <!-- define global column types for dbms -->
+    <property global="true" dbms="postgresql" name="id.type" value="text"/>
+    <property global="true" dbms="postgresql" name="text.type" value="text"/>
+    <property global="true" dbms="postgresql" name="long.text.type" value="text"/>
+    <property global="true" dbms="postgresql" name="text.array.type" value="text[]"/>
+    <property global="true" dbms="postgresql" name="jsonb.type" value="jsonb"/>
+    <property global="true" dbms="mariadb,mysql" name="id.type" value="varchar(50)"/>
+    <property global="true" dbms="mariadb,mysql" name="text.type" value="varchar(255)"/>
+    <property global="true" dbms="mariadb,mysql" name="long.text.type" value="longtext"/>
+    <property global="true" dbms="mariadb,mysql" name="text.array.type" value="longtext"/>
+    <property global="true" dbms="mariadb,mysql" name="jsonb.type" value="longtext"/>
+    <!-- define changesets postgres-only properties not used-->
     <include file="changesets/20230410_schema_reset.yaml" relativeToChangelogFile="true"/>
     <include file="changesets/20230530_avoid_postgres_specific_features.yaml" relativeToChangelogFile="true"/>
+    <include file="changesets/20230606_postgres_rename_columns_and_constraints.yaml" relativeToChangelogFile="true"/>
+    <!-- define changesets common-schema properties are used but dbms: tag set to mariadb,mysql-->
+    <include file="changesets/20230606_schema_common_reset.yaml" relativeToChangelogFile="true"/>
+    <include file="changesets/20230606_schema_common_extend.yaml" relativeToChangelogFile="true"/>
```

Comment:
What do you think about combining these 2 schema files? Given that we'll only deploy this to a fresh DB (now in mariadb/mysql, hopefully soon in postgres), I don't think there's any need to add the text[], jsonb columns + notnull constraints, only to immediately drop them.